index.html

```diff
@@ -13,12 +13,25 @@
     <div class="notif-bar">
       <a href="index.html"><h4>God-Logic</h4></a>
       <nav><img src="images/hamburger.png" alt="hamburger" class="hamburger"></nav>
+       <ul>
+      <a href="">Portfolio</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+      <img src="images/Msg-icon.png" alt="message-icon" class="message-icon">
+    </ul>
```

Comment:
- [ ] For the Background image in your headline section one the desktop, kindly adjust the image to start from the top margin. Though might feel this suite you, the idea is to stick to requirements as much as possible, just as it will be for real clients 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/228417874-6b78d577-9fc8-4856-b53f-d95d6c2a18bf.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/228417911-9bece623-dcba-4c9f-a7ee-ff7c5776505e.png)